velox/exec/HashAggregation.cpp

```diff
@@ -414,6 +414,13 @@ RowVectorPtr HashAggregation::getOutput() {
     if (!newDistincts_) {
       if (noMoreInput_) {
         finished_ = true;
+        if (groupingSet_->hasDefaultGlobalGroupingSetOutput()) {
+          prepareOutput(groupingSet_->numGlobalGroupingSets());
+          VELOX_CHECK(groupingSet_->getDefaultGlobalGroupingSetOutput(
```

Comment:
It might be more readable (and safer) to make the call to getDefaultGlobalGroupingSetOutput outside of the check. As a general guideline, it is better to write code so that checks can be safely removed. Here, removing the check will change the logic.